spec/forms/additional_claimants_form_spec.rb

```diff
@@ -77,7 +77,7 @@
         additional_claimants_form.save
         claim.secondary_claimants.reload
 
-        attributes[:collection_attributes].each_with_index do |(_, attributes), index|
+        attributes[:collection_attributes].sort.each_with_index do |(_, attributes), index|
```

Comment:
Is this sort necessary? 

Same on line 99 👇

Reply:
@leoapost this is where the test was randomly failing. I think this is a fix for that.